pom.xml

```diff
@@ -50,7 +50,7 @@
     <commonstext.version>1.11.0</commonstext.version>
     <fasterxml.version>2.17.2</fasterxml.version>
     <guava.version>32.0.1-jre</guava.version>
-    <hadoop.version>3.3.6</hadoop.version>
+    <hadoop.version>3.4.0</hadoop.version>
```

Comment:
Here we need to analyze the dependency tree with `mvn dependency:tree`, see the diff vs the previous version, and exclude the duplicates.

Reply:
yes, thats what I was thinking..